lib/menu.rb

```diff
@@ -0,0 +1,34 @@
+require_relative 'dish'
+
+class Menu
+  
+  def initialize
+    @menu = [
+      Dish.new("Cookies & Cream", 5.0),
+      Dish.new("Chocolate", 4.3),
```

Comment:
What about maybe having the hash inside spec class and calling from it inside the tests. This will make it easier to update for future prices and additions to the menu.